src/intercomparison.py

```diff
@@ -0,0 +1,67 @@
+import sklearn.metrics
+import geopandas as gpd
+import pandas as pd
+import rasterio as rio
+from pyproj import Proj
+from pyproj import transform
+import numpy as np
+import json
+import os
+
+def run_comparison(validationPath: str, cropmapPath: str, validation_projection = None, cropmap_projection: str = None, asJSON = False, subset: str = None):
+    
+  # Assume datatypes to be csv and tiff for validation and cropmap respectively
+  report = []
+  cropmap = rio.open(cropmapPath)
+  validation = gpd.read_file(validationPath)
+
+  # Resets csv to specified subset
+  if subset != None:
+    validation = validation[validation['subset'] == subset]
```

Comment:
Processing the china tiffs required the entire csv since it had no subset. The function should default to validation but I added this in case it was needed again.

Reply:
After looking over it, I added this section so that I could specifically use the testing subset to match my outputs with [Ryan's analysis ](https://storymaps.arcgis.com/stories/d3fa590f11c845c8ba4ee6c8fbb2bf2d)